cypress/integration/pages/featureIndexPage/testsForAMPOnly.js

```diff
@@ -1,15 +1,17 @@
+import runAMPAdsTests from '../../../support/helpers/adsTests/testsForAMPOnly';
+
 // For testing important features that differ between services, e.g. Timestamps.
 // We recommend using inline conditional logic to limit tests to services which differ.
 export const testsThatAlwaysRunForAMPOnly = ({ service, pageType }) => {
   describe(`No testsToAlwaysRunForAMPOnly to run for ${service} ${pageType}`, () => {});
 };
 
 // For testing features that may differ across services but share a common logic e.g. translated strings.
-export const testsThatFollowSmokeTestConfigForAMPOnly = ({
-  service,
-  pageType,
-}) =>
-  describe(`No testsThatFollowSmokeTestConfigForAMPOnly for ${service} ${pageType}`, () => {});
+export const testsThatFollowSmokeTestConfigForAMPOnly = ({ service }) => {
+  if (Cypress.env('APP_ENV') === 'local') {
```

Comment:
Just making sure I understand, our smoke tests on CI will run with the `APP_ENV` `local`? If so that's good as this test will have loads of value.

Reply:
It will only run on PRs - not on the Test / Live E2Es (because overriding the headers doesn't seem to work, you would need to be on a proxy)